### ISSUE TITLE
TypeErrors when I try to load the calendar

### ISSUE THREAD

**Issue by risafj** [nathanreyes/v-calendar#354]
I have a Vue app written with Typescript using `vue-class-component`. I'm trying to use `v-calendar` in my single file component (`calendar.vue`), and then render that in a page (`schedule.vue`), but I get the following two errors when I try loading that page. Everything else in that page gets rendered, but  there is a tiny dot where the calendar should be for some reason.

` Error in created hook: "TypeError: Cannot read property 'MMMM YYYY' of undefined"`
`TypeError: Cannot read property 'prevMonthComps' of null`

This is my `calendar.vue`:
```
<template lang="pug">
  Calendar
</template>

<script lang="ts">
import 'v-calendar/lib/v-calendar.min.css';
import { Calendar, setupCalendar } from 'v-calendar';
import { Component, Vue } from 'vue-property-decorator';

@Component({
  components: {
    Calendar,
    setupCalendar
  }
})

export default class CalendarComponent extends Vue {
  data() {
    return {
      attrs: [
        {
          key: 'today',
          highlight: {
            backgroundColor: '#ff8080',
          },
          dates: new Date(2018, 0, 1)
        }
      ],
    };
  }
  mounted():any {
    setupCalendar({
      firstDayOfWeek: 2,
    });
  }
}
</script>

```

And my condensed `schedule.vue`:
```
<template lang="pug">
  h1. Schedule 
  CalendarComponent
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';
import CalendarComponent from '../components/calendar.vue';

@Component({
  components: {
    CalendarComponent
  }
})

export default class Schedule extends Vue {
}
</script>
```

My questions are:
1. Is `v-calendar` incompatible with something I'm doing, like using Typescript, or `vue-class-components`?
2. Or am I configuring something wrong?

There seems to be little info aside from [this](https://github.com/nathanreyes/v-calendar#3-reference-in-your-component-templates) about using this module in a component.

**Comment by nathanreyes** [comment]
Hi @risafj. Are you using using the `@next` distribution? If so, `setupCalendar()` requires you to pass in the `Vue` instance as the first argument and the options object as the second argument.

This was an error in the documentation on my part and was addressed per #380

**Comment by risafj** [comment]
Thanks for the response. Unfortunately, I ultimately didn't use v-calendar for my project, so you can close this issue now.
